Examples/Scripts/FreeSurferPipelineBatch-long.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+function identify_timepoints
+{
+    local subject=$1
+    local tplist=""
+    local tp visit n
+
+    #build the list of timepoints (sessions)
+    n=0
+    for session in ${PossibleVisits[*]}; do
+        tp="${subject}_${session}"
+        if [ -d "$StudyFolder/$tp" ] && ! [[ " ${ExcludeVisits[*]} " =~ [[:space:]]"$tp"[[:space:]] ]]; then
+             if (( n==0 )); then 
+                    tplist="$tp"
+             else
+                    tplist="$tplist,$tp"
+             fi
+        fi
+        ((n++))
+    done
+    echo $tplist
+}
+
+get_usage_and_exit(){
+    echo "usage: "
+    echo "FreeSurferPipelineBatch-long.sh [options]"
+    echo "options:"
+    echo "  --runlocal                      run locally [FALSE]"
+    exit -1
+}
+
+command_line_specified_run_local=FALSE
+while [ -n "$1" ]; do
+    case "$1" in
+        --runlocal) shift; command_line_specified_run_local=TRUE ;;
+        *) shift ;;
+    esac
+done
+
+#################################################################################################
+# General input variables
+##################################################################################################
+#Location of Subject folders (named by subjectID)
+StudyFolder="/my/study/path"
+#The list of subject labels, space separated
+Subjects=(HCA6002236 HCA6002237 HCA6002238)
+#The list of possible visits (aka timepoints, sessions) that each subject may have. Timepoint directories should be named <Subject>_<Visit>.
+PossibleVisits=(V1_MR V2_MR V3_MR V4_MR V5_MR V6_MR V7_MR V8_MR V9_MR V10_MR)
+#The list of possible visits that each subject may have. Timepoint (visit) is expected to be named <Subject>_<Visit>.
+#Actual visits (timepoints) are determined based on existing directories that match the visit name pattern.
+ExcludeVisits=(HCA6002237_V1_MR HCA6002238_V1_MR)
```

Comment:
Since these are not real session identifiers, they should be more obviously fake.  Can also leave the example commented out and provide an empty array.
```suggestion
#ExcludeVisits=(HCA1234567_V1_MR HCA7654321_V1_MR)
ExcludeVisits=()
```